scenes/CanvasLayer.gd

```diff
@@ -0,0 +1,20 @@
+extends CanvasLayer
+
+@onready var inventory = $InventoryGui
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	inventory.close() # Replace with function body.
```

Comment:
can toggle inventory to be auto hidden on level 1 scene so don't need this inventory.close

Reply:
ima keep this in for now i might change it in a later branch